.github/workflows/build.yml

```diff
@@ -105,8 +106,23 @@ jobs:
           img="$(jq <<<"$json" -r '.build.img')"
           if crane digest "$img"; then
             echo >&2 "error: '$img' already exists! cowardly refusing to overwrite it"
-            exit 1
+            echo 'cancel=exists' >> "$GITHUB_OUTPUT"
+          else
+            echo 'cancel=' >> "$GITHUB_OUTPUT"
           fi
+        id: check
+      - name: Cancel If Built
+        if: steps.check.outputs.cancel == 'exists'
+        uses: andymckay/cancel-action@435124153eb37d6a62a29d053a7e449652f89d51 # https://github.com/andymckay/cancel-action/commits/HEAD
```

Comment:
Are these changes worth not using the latest "release"?

https://github.com/andymckay/cancel-action/compare/0.3...master

Reply:
Heh, this was actually more about pinning to an explicit commit than it was avoiding a release -- I'm sure Andy is great, but it seemed prudent to make sure the action we actually use is the one whose code I reviewed.  I'm happy to adjust this pin down to the commit of `0.3` if you think that's better, but I don't feel very strongly about it either way (and don't expect many changes in that module in the future).

We could instead invoke `curl` against the API directly, but it felt better to lean on someone else who's already done that, especially since their pre-built action for it is already really minimal.